streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetrics.java

```diff
@@ -96,6 +103,22 @@ public static Sensor processLatencySensor(final String threadId,
         return emptySensor(threadId, taskId, PROCESS_LATENCY, RecordingLevel.DEBUG, streamsMetrics);
     }
 
+    public static Sensor activeProcessRatioSensor(final String threadId,
+                                                  final String taskId,
+                                                  final StreamsMetricsImpl streamsMetrics) {
+        final String name = ACTIVE_TASK_PREFIX + PROCESS + RATIO_SUFFIX;
+        final Sensor sensor = streamsMetrics.taskLevelSensor(threadId, taskId, name, Sensor.RecordingLevel.INFO);
+        sensor.add(
+            new MetricName(
+                name,
+                TASK_LEVEL_GROUP,
+                PROCESS_RATIO_DESCRIPTION,
+                streamsMetrics.taskLevelTagMap(threadId, taskId)),
+            new Value()
+        );
```

Comment:
```suggestion
        addValueMetricToSensor(
            sensor,
            TASK_LEVEL_GROUP,
            streamsMetrics.taskLevelTagMap(threadId, taskId),
            name,
            PROCESS_RATIO_DESCRIPTION
        );
```
req: Please use `StreamsMetricsImpl#addValueMetricToSensor()`

Reply:
ack.